lib/omnibus/generator_files/Gemfile.erb

```diff
@@ -13,9 +13,10 @@ gem 'omnibus', '~> <%= Omnibus::VERSION.split('.')[0...-1].join('.') %>'
 # by running `bundle install --without development` to speed up build times.
 group :development do
```

Comment:
I wonder if we should do away with this `:development` group and recommend people use ChefDK?